packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -2,9 +2,11 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Dialog, ButtonToggle } from '@deriv/components';
 import { localize } from 'Components/i18next';
-import { BuySellTable } from './buy-sell-table.jsx';
+// import { BuySellTable } from './buy-sell-table.jsx';
```

Comment:
Do you need to keep this comment? (if yes, please add TODO)

Reply:
No need. Thanks 👍